communication_irl_doc.Rmd

```diff
@@ -207,13 +207,13 @@ Thank you for attending our `r params$workshop` workshop this week. We hope that
 
 If you haven’t already, please be sure to complete our (post-workshop survey)[post-workshop survey] [1]. Your answers are essential to allow us to continuously improve our workshops for future learners.  
   
-You will shortly be able to find all the notebooks and collaborative documents from the workshop in [the files directory of the workshop repository](`r paste0(params$repo, "/tree/main/files")`) [2]. All the lesson material used in the lesson can be found `r params$lesson_url_ref` [3].
+You will shortly be able to find all the notebooks and collaborative documents from the workshop in [the files directory of the workshop repository](`r paste0(params$repo, "/tree/main/files")`) [2]. All the lesson material used in the lesson can be found [here](`r params$lesson_url_ref`) [3].
```

Comment:
does the `r params$lesson_url_ref` point to the rendered version of the lesson?

Reply:
yes, that's correct. The link to the repo is `params$repo`.